pkgs/development/tools/database/atlas/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, git
+}:
+
+buildGoModule rec {
+  pname = "atlas";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "ariga";
+    repo = pname;
```

Comment:
`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21